File: pset6/mario/more/mario.py
from cs50 import get_int

height = get_int("Height: ")
# Check the correct input
while height < 0 or height > 23:
    height = get_int("Height: ")
# i - rows counter, s- spaces counter, b && d - sharps counter
for i in range(height):
    for s in range(height - i - 1):
        print(' ', end="")
    for d in range(i + 1):
        print("#", end="")
    print("  ", end="")
    for b in range(i + 1):
        print('#', end="")
    print('')

File: pset1/mario/less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    //Check the correct input
    do
    {
        height = get_int("Height:");
    }
    while (height < 0 || height > 23);
    //i - rows counter, s- spaces counter, b - sharps counter
    for (int i = 0; i < height; ++i)
    {
        for (int s = 1; s < height - i; ++s) //Number of spaces = height - 1 - row
        {
            printf(" ");
        }
        for (int b = 0; b <= i + 1; ++b) //Number of sharps = row + 1
        {
            printf("#");
        }
        printf("\n");
    }
}